concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -160,6 +167,13 @@ resources:
       uri: https://github.com/cloudfoundry/cf-release
       tag_filter: {{cf-release-version}}
 
+  - name: vpc-terraform-state
```

Comment:
There is already a `vpc-tfstate` resource. Why create a new resource pointing to the same file?